### PR TITLE
BugFix Pull Request

### PR DESCRIPTION
**Pull request by shenditya**
Addresses some issue:
-Setting Page Error
-Gambar tidak muncul di dashboard page dan detail kegiatan/berita
-field pekerjaan keyboard nya numeric harusnya alphabet
-Fix border profile & address page
-Hilde icon picker on profile page
-Backbutton address page
-Logout Button

### CODE REVIEW

File: Mitra Farmers/Mitra Farmers/Model/Request/Profile/ProfileResponse.swift [hunk 1]
```diff
@@ -8,7 +8,7 @@
 import Foundation
 
 class ProfileResponse: Codable {
-    let id: Int
+   // let id: Int
```
Comment (maskalsaintek) [inline Mitra Farmers/Mitra Farmers/Model/Request/Profile/ProfileResponse.swift:11]: remove comment

File: Mitra Farmers/Mitra Farmers/Model/Request/Profile/ProfileResponse.swift [hunk 2]
```diff
@@ -21,10 +21,11 @@ class ProfileResponse: Codable {
     let kelurahan: String
     let postalCode: String
     let profilePicture: ProfilePictureResponse
-    let status: Bool
+    let status: Int
     
     enum CodingKeys: String, CodingKey {
-        case id, name, email, kecamatan, kelurahan, status
+        //case id
```
Comment (maskalsaintek) [inline Mitra Farmers/Mitra Farmers/Model/Request/Profile/ProfileResponse.swift:27]: ditto

File: Mitra Farmers/Mitra Farmers/Scene/Summary/FarmerSummaryViewController.swift [hunk 3]
```diff
@@ -60,6 +57,18 @@ class FarmerSummaryViewController: UIViewController {
...
         
         tableView.register(UINib(nibName: "FarmerSummaryCell", bundle: nil), forCellReuseIdentifier: "FarmerSummaryCell")
+        
+        guard let profile = Preference.profile, (profile.status == 1) else {
+            return
+        }
+        
+        for frameView in frameViews {
+            frameView.isHidden = profile.status == 1
+            frameViewsHeight.constant = profile.status == 1 ? 0 : 179
```
Comment (maskalsaintek) [inline Mitra Farmers/Mitra Farmers/Scene/Summary/FarmerSummaryViewController.swift:67]: better make uistackview sit tap utk skrg app

**Review by maskalsaintek — APPROVED** [review]

File: Mitra Farmers/Mitra Farmers/Scene/Summary/FarmerSummaryViewController.swift (hunk 3 above)
Comment (shenditya) [reply Mitra Farmers/Mitra Farmers/Scene/Summary/FarmerSummaryViewController.swift:67]: oke mas, ku ganti pake stack

**Review by maskalsaintek — APPROVED** [review]

**Pull request merged**